package.json

```diff
@@ -4,7 +4,7 @@
   "version": "9.1.1",
   "author": "Adobe Inc.",
   "bin": {
-    "aio": "./bin/run"
```

Comment:
I'm sure this won't affect things, but I wouldn't take the risk of updating this here. Should be a separate issue.

Reply:
it was updated automatically when i ran npm i. Will revert it.

Reply:
reverted